crates/optimism/node/src/node.rs

```diff
@@ -158,7 +161,7 @@ impl OpNode {
         ComponentsBuilder::default()
             .node_types::<Node>()
             .pool(OpPoolBuilder::default())
-            .payload(OpPayloadBuilder::new(compute_pending_block))
+            .payload(OpPayloadBuilder::new(compute_pending_block, OpDAConfig::default()))
```

Comment:
I see,

we now need to change this `components` function and make it take &self, just like:

https://github.com/paradigmxyz/reth/blob/02f76b813ed27a97fd4b90c1a46e1c1cf0cbb134/crates/optimism/node/src/node.rs#L138-L138

because otherwise we use two different `DaConfig`

https://github.com/paradigmxyz/reth/blob/02f76b813ed27a97fd4b90c1a46e1c1cf0cbb134/crates/optimism/node/src/node.rs#L201-L201

Reply:
Changed stuff up, lmk if im off anywhere.